src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/auth/AuthApiService.kt

```diff
@@ -71,8 +75,31 @@ class AuthApiService(
     usernames,
     EmailList::class.java,
   )
+
+  fun findUsers(
+    name: String?,
+    status: Status?,
+    authSources: List<AuthSource>?,
+    page: Int?,
+    size: Int?,
+    sort: String?,
+  ) =
+    userWebClientUtils.getWithParams(
+      "/api/user/search",
+      object : ParameterizedTypeReference<PagedResponse<ExternalUserDetailsDto>>() {},
+      mapNonNull(
```

Comment:
Added mapNonNull - not sure if we need it but if the endpoint isn't called with it, it's not passed on. Possibly discuss?